### ISSUE TITLE
I propose this repo gets archived

### ISSUE THREAD

**Issue by abmusse** [KaltunAhmed/prayer-times#72]
I propose this repo gets archived due to the breakage of the location API.

I don't think we have any viable alternatives for a free API at the moment.

This was a fun project and appreciate working with you all @Nuh-h @KaltunAhmed.

Perhaps we may work together on another project in the future insha Allah!

_Originally posted by @abmusse in https://github.com/KaltunAhmed/prayer-times/issues/65#issuecomment-2196026999_

**Comment by Nuh-h** [comment]
I agree. Likewise, @abmusse and @KaltunAhmed - it was a pleasure to work with you on this, and definitely open to collaborating soon!

I'll defer to @KaltunAhmed for actioning it or any other ideas.

**Comment by KaltunAhmed** [comment]
Thank you both for taking part in this project. I think it would be a good idea to Archive it!

May Allah reward you both for your efforts
